go.mod

```diff
@@ -9,7 +9,9 @@ require (
 
 require (
 	github.com/golang/protobuf v1.3.2 // indirect
-	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110 // indirect
+	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
```

Comment:
Please revert all changes to `go.mod` and `go.sum` as they are unrelated to this PR.